quantumdb-core/pom.xml

```diff
@@ -8,7 +8,7 @@
 	<parent>
 		<groupId>io.quantumdb</groupId>
 		<artifactId>quantumdb</artifactId>
-		<version>0.4.1-SNAPSHOT</version>
+		<version>0.4.2-SNAPSHOT</version>
```

Comment:
No need to do this, this will be done by the release process. Applies to all the POM files.